etcdmain/etcd.go

```diff
@@ -333,35 +335,28 @@ func startProxy(cfg *config) error {
 	return nil
 }
 
-// setupCluster sets up an initial cluster definition for bootstrap or discovery.
-func setupCluster(cfg *config) (*etcdserver.Cluster, error) {
-	var cls *etcdserver.Cluster
-	var err error
+// setupPeerURLsMapAndToken sets up an initial peer URLsMap and cluster token for bootstrap or discovery.
```

Comment:
can we change the name to `getPeerURLs...`. It does not set up anything but simply return them now. (yea. it used to set up a cluster struct)